packages/manager/apps/web/client/app/domain/zone/record/domain-zone-record.controller.js

```diff
@@ -700,7 +702,9 @@ angular.module('App').controller(
 
     useSpfOvh() {
       this.model.target = {};
-      this.model.target.include = 'mx.ovh.com';
+      this.model.target.include = `mx.ovh.${
+        Environment.getRegion() === CANADA_REGION ? 'ca' : 'com'
+      }`;
```

Comment:
```suggestion
      this.model.target.include = Environment.getRegion() === 'CA' ? 'mx.ovh.ca' : 'mx.ovh.com';
```

As it seems to be an hotfix, I think we can get rid of refactor everything to use a constant and use `'CA'`.
The refacto should be done in another PR.

Reply:
Ok got it.